sig-architecture/api-review-process.md

```diff
@@ -27,39 +27,37 @@ Ideally, those whose API review priority is shifted in a release-impacting way w
 
 # What APIs need to be reviewed?
 
-* What are the kind of reviews?
+## Mandatory
+The changer is expected to submit the changes to API review, and the change is blocked on the API reviewer approval.  As a general rule of thumb, anything that is considered to be part of "core Kubernetes" requires a mandatory API review.
```

Comment:
I think the terminology of "core" here might be the issue.  We could change this as `part of Kubernetes` and then better describe `sig-owned Kubernetes ecosystem components`.